doc/rtd/reference/network-config-format-v1.rst

```diff
@@ -251,9 +251,10 @@ Nameserver
 Users can specify a ``nameserver`` type. Nameserver dictionaries include
 the following keys:
 
-- ``address``: List of IPv4 or IPv6 address of nameservers.
-- ``search``: List of hostnames to include in the :file:`resolv.conf` search
-  path.
+- ``address``: List of IPv4 or IPv6 address of nameservers or string with
+  single nameserver.
```

Comment:
nit: documenting every special case for multiple types may be more complete, but the relative benefit to users for supporting multiple types seems pretty low relative to the benefit since it often leads to unclarity - a user might interpret "list or string of IPv4 address of nameservers" to mean the following could be valid:

`"1.1.1.1 8.8.8.8 4.4.4.4"`

I agree that when our schema is wrong, we should fix it to be correct, however I lean towards a leaner "one way to do it" preference for future features visible in the UI - and documentation that reflects that mentality as well.